### PR TITLE
PDF generation & Zipexport

### PR DESCRIPTION
**Pull request by shylux**
screen shot 2015-04-15 at 09 55 31
screen shot 2015-04-15 at 09 55 43

@jone

### CODE REVIEW

File: ftw/blog/locales/de/LC_MESSAGES/ftw.blog.po [hunk 1]
```diff
@@ -157,3 +157,15 @@ msgstr "Galerie"
...
+msgid "latex_categories"
+msgstr "Kategorien"
+
+msgid "latex_tags"
+msgstr "Tags"
+
+#. Default: "Unpublished"
+#: ./ftw/blog/latex/views.py:82
+msgid "latex_unpublished"
+msgstr "Unpubliziert"
```
Comment (maethu) [inline ftw/blog/locales/de/LC_MESSAGES/ftw.blog.po]: @shylux "Nicht publiziert" would be better.

**Comment by maethu** [comment]
@shylux Please also add a zip_export extra and one line of documentation in the readme.

**Comment by shylux** [comment]
@maethu updated

**Comment by shylux** [comment]
@maethu now with `extras_require`

**Comment by maethu** [comment]
@shylux Looks good :smile:

**Pull request merged**